projects/js-packages/publicize-components/src/components/form/index.tsx

```diff
@@ -8,90 +8,45 @@
 
 import { Disabled, PanelRow } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
-import { Fragment, useMemo } from '@wordpress/element';
+import { Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { usePublicizeConfig } from '../../..';
-import useAttachedMedia from '../../hooks/use-attached-media';
-import useDismissNotice from '../../hooks/use-dismiss-notice';
-import useFeaturedImage from '../../hooks/use-featured-image';
-import useImageGeneratorConfig from '../../hooks/use-image-generator-config';
-import useMediaDetails from '../../hooks/use-media-details';
-import useMediaRestrictions, { NO_MEDIA_ERROR } from '../../hooks/use-media-restrictions';
-import useRefreshAutoConversionSettings from '../../hooks/use-refresh-auto-conversion-settings';
 import useSocialMediaConnections from '../../hooks/use-social-media-connections';
 import { store as socialStore } from '../../social-store';
 import { ThemedConnectionsModal as ManageConnectionsModal } from '../manage-connections-modal';
 import { AdvancedPlanNudge } from './advanced-plan-nudge';
-import { AutoConversionNotice } from './auto-conversion-notice';
 import { BrokenConnectionsNotice } from './broken-connections-notice';
 import { ConnectionsList } from './connections-list';
-import { InstagramNoMediaNotice } from './instagram-no-media-notice';
+import { MediaValidationNotices } from './media-validation-notices';
 import { SettingsButton } from './settings-button';
 import { ShareCountInfo } from './share-count-info';
 import { SharePostForm } from './share-post-form';
 import styles from './styles.module.scss';
 import { UnsupportedConnectionsNotice } from './unsupported-connections-notice';
-import { ValidationNotice } from './validation-notice';
 
 /**
  * The Publicize form component. It contains the connection list, and the message box.
  *
  * @returns {object} - Publicize form component.
  */
 export default function PublicizeForm() {
-	const { connections, hasConnections, hasEnabledConnections } = useSocialMediaConnections();
-	const { isEnabled: isSocialImageGeneratorEnabledForPost } = useImageGeneratorConfig();
-	const { shouldShowNotice, NOTICES } = useDismissNotice();
+	const { hasConnections, hasEnabledConnections } = useSocialMediaConnections();
 	const {
 		isPublicizeEnabled,
 		isPublicizeDisabledBySitePlan,
 		needsUserConnection,
 		userConnectionUrl,
 	} = usePublicizeConfig();
 
-	const { numberOfSharesRemaining, useAdminUiV1 } = useSelect( select => {
+	const { useAdminUiV1 } = useSelect( select => {
 		const store = select( socialStore );
 		return {
-			numberOfSharesRemaining: store.numberOfSharesRemaining(),
 			useAdminUiV1: store.useAdminUiV1(),
 		};
 	}, [] );
 
 	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
 
-	const isAutoConversionEnabled = useSelect(
```

Comment:
I think we will still need this. With this removed even with auto-conversion turned off, I won't see a notice for invalid media that are convertible. This means that for folks who have it off, they won't see the error with this change but the shares will fail